### ISSUE TITLE
Is this the approach to be used for angular4 as well?

### ISSUE THREAD

**Issue by ganeshmuthuvelu** [nickgoodrow/SummernoteNg2_WorkAround#1]
Is this the approach to be used for angular4 as well? or is there any other stable way to use this in an angular4 application?

**Comment by nickgaudreau** [comment]
not really... i wrote this a while back when learning ng2. I'd suggest to use summernote as a service with the help of InjectionToken, declare into your app.module provider, then inject wherever you need. The api is small so you can even implement it as an interface to have the intellisense in your code but not necessary. 
You can do the same with Jquery without the interface of course. 

What I just explained above would be the proper way to use it in the sense of:  without breaking the ng DI, and without using global.
